driver/vhci/vhci_read.c

```diff
@@ -101,12 +101,13 @@ store_urb_reset_pipe(PIRP irp, PURB urb, struct urb_req *urbr)
 		return STATUS_INVALID_PARAMETER;
 	}
 
-	set_cmd_submit_usbip_header(hdr, urbr->seq_num, urbr->vpdo->devid, in, urb_rp->PipeHandle, 0, 0);
+	set_cmd_submit_usbip_header(hdr, urbr->seq_num, urbr->vpdo->devid, 0, 0, 0, 0);
```

Comment:
We want to use default Control pipe

Reply:
I think that "in" parameter cannot be dropped, which indicates direction at usbip protocol level.

Reply:
Yes but `in` is initialized from PipeHandle. But we cannot send packet to PipeHandle but to default Control Pipe (from Host To device - which is zero).
PipeHandle is present only for identification of endpoint type.

Or  I am wrong? (Maybe I do not understand USBIP protocol enaught)

Reply:
@koudis : Oh, I see. You mean that this control command should be always OUT(from host to device). You're right. I'll modify the recent commit.

Reply:
@cezanne  exactly!